src/cli.ts

```diff
@@ -180,13 +189,14 @@ axios.interceptors.request.use(config => {
 process.on('unhandledRejection', onError)
 
 const start = async () => {
+  const initTimeStartTime = process.hrtime()
```

Comment:
This should be the first thing in this file, so we include the time to import all libs.

This way it wouldn't be necessary to modify `main`'s arguments

Reply:
Just after: `#!/usr/bin/env node`